direct/src/filter/CommonFilters.py

```diff
@@ -15,7 +15,9 @@
 # However, until I write some more filters myself, I won't know what
 # that framework is.  Until then, I'll settle for this
 # clunky approach.  - Josh
+from panda3d.core import VirtualFileSystem
 
+vfs: VirtualFileSystem = VirtualFileSystem.get_global_ptr()
```

Comment:
This type annotation is only useful if the type-checker doesn't have information about what `VirtualFileSystem.get_global_ptr` returns (i.e., through stubs). If that's the case, the type-checker almost certainly doesn't know anything else about a `VirtualFileSystem`, so it doesn't really do much in any case. I'd recommend removing it, unless I'm missing something.

Reply:
Thanks for catching that.